float64[12] contact_force
